setup.py

```diff
@@ -18,7 +18,7 @@
     url='https://github.com/neon-jungle/wagtail-metadata',
 
     install_requires=[
-        'wagtail~=2.0',
+        'wagtail~=3.0',
```

Comment:
If we set the minimum version to 3, we won't need the try/except around the imports. I'm ok with releasing a major version for 3.0 support to no have the old imports hanging around

Reply:
Ah...good catch :) 

Is it that bad to keep old imports ? Wagtail 3.0 was just released and not even LTS.
We could support wagtail 2 until the next LTS is out, and add a warning in changelog about future deprecation ?

Reply:
it's not bad, it's just not super necessary. The current version of this plugin works with any wagtail 2.x, so happy to cut a new release for the next version, a note in the readme wouldn't hurt either

Reply:
Fair enough, I am happy for you to just release a new version then. 
I will update my PR